internal/cmd/beta/network-area/route/describe/describe.go

```diff
@@ -137,6 +138,19 @@ func outputResult(p *print.Printer, outputFormat string, route *iaas.Route) erro
 		table.AddRow("Prefix", *route.Prefix)
 		table.AddSeparator()
 		table.AddRow("Nexthop", *route.Nexthop)
```

Comment:
Btw, could you make all row names capitalized? We have it like that in all other commands, not sure why this one wasn't 😬

Reply:
Like `table.AddRow("PREFIX", *route.Prefix)`